### PR TITLE
chore: add generated docs files to release commit

### PR DESCRIPTION
**Pull request by baileympearson**
#### What is changing?

the `-a` flag is added to `standard-version` in our release script, so that generated documentation will be included in the release commit.

note that this doesn't automatically generate the documentation.  but without this change, any generated files will be ignored by `standard-version` when the release commit is created.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
     "fix:eslint": "npm run check:eslint -- --fix",
     "prepare": "node etc/prepare.js",
     "preview:docs": "ts-node etc/docs/preview.ts",
-    "release": "standard-version -i HISTORY.md",
+    "release": "standard-version a -i HISTORY.md",
```
Comment (nbbeeken) [inline package.json:121]: Needs `-a`
and we chatted a bit offline: we might want a script here that asserts the repo has no other outstanding staged files as this flag could accidentally commit source changes. I would say add a `git reset` here but that could be destructive so something that would just exit failure if any staged changes are outside the docs folder would cover us.

**Review by nbbeeken — CHANGES_REQUESTED** [review]

**Review by dariakp — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nbbeeken) [reply package.json:121]: :jinx:

**Review by nbbeeken — DISMISSED** [review]

File: out.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (dariakp) [inline out.txt:1]: if this is a test file, it seems that the script doesn't quite work as expected

**Review by dariakp — CHANGES_REQUESTED** [review]

File: out.txt (hunk 2 above)
Comment (baileympearson) [reply out.txt]: Yeah, the changes I made this afternoon weren't quite ready for another review it seems 😅

**Comment by baileympearson** [comment]
Here's a (long) video showing the new functionality 

https://drive.google.com/file/d/1Vq88D--WWUpGLm_O8koP2jSTO4HDYyyn/view?usp=sharing

**Review by nbbeeken — APPROVED** [review]

**Review by dariakp — APPROVED** [review]

**Pull request merged**